### DISCUSSION TITLE
transformers   4.29.0 , cuda 12.1, pytorch 2.1 dev,

### DISCUSSION THREAD

**Discussion by adrianpuiu** [seruva19/kubin#48] [General]
(venv) agp@agp-ROG-Strix-G35CG-G35CG:~/kubin$ pip list | grep transformers
transformers             4.29.0

WARNING:root:Pytorch pre-release version 2.1.0.dev20230509+cu121 - assuming intent to test it

CUDNN - 8.9.1

test =>  passed

**Comment by seruva19** [comment]
OK, thanks.
